File: banda/AMarotoChivite-banda/AMarotoChivite-banda-Java/src/test/java/simulacionBandaMusica/factories/TrompetistaFactoryTest.java
package simulacionBandaMusica.factories;

import org.junit.jupiter.api.Test;
import simulacionBandaMusica.models.Trompetista;

import static org.junit.jupiter.api.Assertions.assertNotNull;

class TrompetistaFactoryTest {

    @Test
    void createTrompetistaTest() {
        Trompetista Trompetista = TrompetistaFactory.create();
        // Trompetista no nulo, para ver si se han asignado correctamente los datos
        assertNotNull(Trompetista);
    }
}

File: banda/AMarotoChivite-banda/AMarotoChivite-banda-Kotlin/src/test/kotlin/simulacionBandaMusica/factories/TrompetistaFactoryTest.kt
package simulacionBandaMusica.factories

import org.junit.jupiter.api.Assertions
import org.junit.jupiter.api.Test
import simulacionBandaMusica.models.Musico

internal class TrompetistaFactoryTest {
    @Test
    fun createTrompetistaTest() {
        val Trompetista: Musico.Trompetista = TrompetistaFactory.create()
        // Trompetista no nulo, para ver si se han asignado correctamente los datos
        Assertions.assertNotNull(Trompetista)
    }
}